public/app/percona/shared/core/reducers.ts

```diff
@@ -415,6 +427,31 @@ export const fetchServerSaasHostAction = createAsyncThunk(
     )
 );
 
+const toUserDetailsModel = (res: UserDetailsResponse): UserDetails => ({
+  userId: res.user_id,
+  productTourCompleted: !!res.product_tour_completed,
+});
```

Comment:
I would suggest putting it in utils and userSlice, along with the store and the description of the types, I think should also be taken out separately, because our file with the reducers is getting longer and more inconvenient, if you don't mind

Reply:
Should be good to go, moving the files out of `index.ts` in `reducers` into own files (preferably if someone is modifying it, they should separate it into it's own file) should affect imports, so the usage should stay the same as it is.